src/hitablelist.h

```diff
@@ -55,7 +55,7 @@ bool HitableList::bounding_box(float t0, float t1, AABB &box) const
     else
         box = temp_box;
 
-    for (auto i=1; i<list_size; ++i)
+    for (size_t i=1; i<list_size; ++i)
```

Comment:
MSVC was complaining about `i` being an `int` and `list_size` being an `std::size_t`.